qiskit/backends/ibmq/ibmqprovider.py

```diff
@@ -19,19 +20,24 @@ def __init__(self, token, url='https://quantumexperience.ng.bluemix.net/api',
                  hub=None, group=None, project=None, proxies=None, verify=True):
         super().__init__()
 
+        if any([hub, group, project]):
+            url = "https://q-console-api.mybluemix.net/api/Hubs/{hub}/" + \
+                  "Groups/{group}/Projects/{project}"
+            url = url.format(hub=hub, group=group, project=project)
+            warnings.warn(
+                "Passing hub/group/project is depreciated in qsikit 0.6+"
```

Comment:
Small nit - can you add a `. ` (dot and space) or something similar, for proper displaying of the warning? This line and the next one are actually concatenated when printing, which would result in `...qiskit 0.6+Use the ...` (also can you tweak `qsikit`?)

Reply:
Done.